jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleProjectProperties.java

```diff
@@ -458,13 +458,20 @@ public JibContainerBuilder runPluginExtensions(
       ContainerBuildPlan buildPlan)
       throws JibPluginExtensionException {
     T extraConfig = null;
-    if (extraConfigType.isPresent() && config.getExtraConfiguration().isPresent()) {
-      try {
+    if (config.getExtraConfiguration().isPresent()) {
+      if (!extraConfigType.isPresent()) {
+        log(
```

Comment:
maybe this should be a hard failure?

Reply:
for example if you try to specify `jib.to.potato` I think you get a failure?

Reply:
Makes sense. Done.